app/vmalert/README.md

```diff
@@ -70,8 +73,20 @@ Usage of vmalert:
         Address to listen for http connections (default ":8880")
   -notifier.url string
         Prometheus alertmanager URL. Required parameter. e.g. http://127.0.0.1:9093
+  -remotewrite.basicAuth.password string
+        Optional basic auth password for -remotewrite.url
+  -remotewrite.basicAuth.username string
+        Optional basic auth username for -remotewrite.url
   -remotewrite.url string
         Optional URL to remote-write compatible storage where to write timeseriesbased on active alerts. E.g. http://127.0.0.1:8428
+  -remoteread.basicAuth.password string
+        Optional basic auth password for -remotewrite.url
+  -remoteread.basicAuth.username string
+        Optional basic auth username for -remotewrite.url
+  -remoteread.lookback duration
+        Lookback defines how far to look into past for alerts metrics.For example, if lookback=1h then range from now() to now()-1h will be scanned. (default 1h0m0s)
+  -remoteread.url remotewrite.url
```

Comment:
It looks like you put `remotewrite.url` inside `-remoteread.url` description into backticks, so it substituted `string` type. Remove backticks in the description.